runners/cromwell_on_google/monitoring_tools/get_yaml_value.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+
+# Copyright 2017 Google Inc.
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+# get_yaml_value.py
+#
+# Utility script for extracting values from YAML.
+# This will typically be called from shell scripts, which typically
+# have fairly hacky ways of extracting values from YAML.
+#
+# An example usage would be where a shell script has YAML output from
+# a gcloud command for an genomics operation and needs to extract fields:
+#
+#  OP=$(gcloud --format=yaml alpha genomics operations describe "${OP_ID}")
+#  CTIME=$(python tools/get_yaml_value.py "${OP}" "metadata.createTime")
```

Comment:
Here are two more references to the "tools" directory. These can both just be removed.
There is nothing in this example that needs to imply that you are in the cromwell_on_google directory:

    CTIME=$(python get_yaml_value.py "${OP}" "metadata.createTime")
    ETIME=$(python get_yaml_value.py "${OP}" "metadata.endTime")